src/main/java/org/traccar/model/Position.java

```diff
@@ -142,6 +142,7 @@
     public static final String ALARM_TAMPERING = "tampering";
     public static final String ALARM_REMOVING = "removing";
 
+
```

Comment:
Looks like there are no changes in this file. Can you please revert this.

Reply:
Removed that.